src/Actions/Action.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace TestMonitor\ActiveCampaign\Actions;
+
+/**
+ * Class Action.
+ *
+ * Defines methods implemented in MakeHttpRequest and ActiveCampaign that used in traits.
+ */
+trait Action
```

Comment:
Interesting solution, I've got 2 remarks though:

- While your new classes make use of this trait, the existing classes won't. This causes some sort of generational problem. I would suggest using this trait in this other classes as well.
- Our Trait naming scheme uses a "verb" - "object" combination. "Use Action" would deviate from this. I'd suggest something like ImplementsActions?

Reply:
Done